plone-6.0.x.cfg

```diff
@@ -25,3 +23,7 @@ cffi = 1.14.6
 # Requirement of flake8>=2.4.0: importlib-metadata<4.3
 # Error: The requirement ('importlib-metadata<4.3') is not allowed by your [versions] constraint (4.11.2)
 importlib-metadata = 4.2.0
+
+# Requirement of Markdown>=2.6.5: importlib-metadata>=4.4
+# Error: The requirement ('importlib-metadata>=4.4') is not allowed by your [versions] constraint (4.2.0)
+Markdown = 3.3.4
```

Comment:
This seems wrong:

- We pin `importlib-metadata = 4.2.0` a few lines above.
- The comment says `Markdown>=2.6.5` requires at least `importlib-metadata` 4.4.
- Yet we pin `Markdown` 3.3.4, which should conflict with the older `importlib-metadata`.

Reply:
Newer versions of `flake8` fix `importlib-metadata<4.3` for Python < 3.8. See:

https://github.com/PyCQA/flake8/blob/5.0.4/setup.cfg#L45

So the pin that exists today:

`importlib-metadata = 4.2.0`

is necessary.

Newer versions of `Markdown` fix `importlib-metadata>=4.4` for Python < 3.10. See:

https://github.com/Python-Markdown/markdown/blob/3.4.1/setup.py#L77

This conflicts. So I pin an older version of `Markdown`, where the `importlib-metadata` constraint didn't exist yet.

Reply:
My point is that the comment suggests that Markdown 2.6.5 already requires the newer importlib-metadata, so I expect to get a version conflict here. I guess it is just buildout that reports this, and gets the 2.6.5 from somewhere. If you change the comment to "Requirement of Markdown 3.4.1+" then I would no longer be confused.

Reply:
Done